tests/test_extensions.py

```diff
@@ -0,0 +1,132 @@
+"""
+Unit tests for Extension Manager
+"""
+
+import sys
+import os
+import unittest
+import mock
+import shutil
+
+dir_of_executable = os.path.dirname(__file__)
```

Comment:
A developer (i.e. me) may want to run a test solely to verify it is passing. 

It's only a line. :)